client/hostinfo_unix.cpp

```diff
@@ -1738,15 +1778,34 @@ int HOST_INFO::get_os_info() {
             }
             strip_whitespace(buf2);
         }
-        strcat(buf2, " [");
-        safe_strcat(buf2, os_version);
-        strcat(buf2, "]");
+        os_version_extra = (string)os_version;
         safe_strcpy(os_version, buf2);
         if (strlen(dist_name)) {
             strcat(os_name, " ");
             safe_strcat(os_name, dist_name);
         }
     }
+
+    string libc_version(""), libc_extra_info("");
+    if (!get_libc_version(libc_version, libc_extra_info)) {
+        msg_printf(NULL, MSG_INFO,
+                "[libc detection] gathered: %s, %s", libc_version.c_str(), libc_extra_info.c_str()
```

Comment:
This looks like debug code.

Reply:
It started out to be put since the hostinfo detection only runs on startup this is a nice addition to the default startup messages I think so I left it in and added a comment that it was left on purpose.